src/components/profileEdit/data.ts

```diff
@@ -42,19 +48,22 @@ const passwordData = [
     label: 'Старый пароль',
     type: 'password',
     disabled: true,
-    value: '•••••••••',
+    value: '123qweQWE',
+    name: 'password'
```

Comment:
```suggestion
    name: 'password',
```
Выше ты ставил тут `,`. Это полезно делать, потому что положительно сказывается на диффах в системах контроля версий